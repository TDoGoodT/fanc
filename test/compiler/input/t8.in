int decrement(int a) {
    return a - 1;
}
int subtract(int a, int ab) {
    int tmpa = a;
    int tmpb = ab;
    while (tmpb > 0) {
        tmpa = decrement(tmpa);
        tmpb = decrement(tmpb);
    }
    return tmpa;
}

void main() {
    int num1 = 10;
    int num2 = 5;

    int result = subtract(num1, num2);

    print("Difference between 10 and 5: ");
    printi(result);
}
