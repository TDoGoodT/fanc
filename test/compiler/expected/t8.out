Difference between 10 and 5: 
5
